src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -48,6 +51,25 @@ public static Index parseIndex(String oneBasedIndex) throws IllegalValueExceptio
     }
 
     /**
+     * Parses {@code args} into an {@code List<Index>} and returns it.
+     * Used for commands that need to parse multiple indexes
+     * @throws IllegalValueException if the specified index is invalid (not non-zero unsigned integer).
+     */
+    public static List<Index> parseMultipleIndexes(String args) throws IllegalValueException {
+        // Example of proper args: " 1 2 3" (has a space in front)
+        List<String> argsList = Arrays.asList(args.split(" "));
+        List<Index> indexList = new ArrayList<>();
+
+        for (String index : argsList) {
+            if (index.isEmpty()) { // An empty space stored as a result of String.split()
```

Comment:
If we split by the regex `"\\s+"`, we won't need to worry about empty indexes :)

Reply:
Hmm, I would still need the if-statement check even with the regex `\\s+` as I want to allow input such as: `fav 1  2   3    4`

Reply:
Thanks Marvin! 
As discussed, this worked:
`List<String> argsList = Arrays.asList(args.trim().split("\\s+"));`